exchange/utils.go

```diff
@@ -94,7 +95,12 @@ func cleanOpenRTBRequests(ctx context.Context,
 	privacyLabels.COPPAEnforced = privacyEnforcement.COPPA
 	privacyLabels.LMTEnforced = lmtEnforcer.ShouldEnforce(unknownBidder)
 
-	if gdpr == 1 {
+	GDPREnabled := accountLevelGDPREnabled(account, labels.RType)
```

Comment:
Nitpick: Variable names within a method should be lower case. `GDPREnabled` -> `gdprEnabled`